### PR TITLE
Added .travis.yml

### PR DESCRIPTION
**Pull request by tkawachi**
I tried to compile the compiler on Travis CI https://travis-ci.org/.
Please check the commands in `.travis.yml`. I'm not absolutely sure about the command (I'm new to Frege!).

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  - sudo apt-get update -qq
+  - sudo apt-get install -y byacc
+
+before_script:
+  - mkdir -p build dist doc
+  # Get the latest fregec.jar to compile the compiler
+  - curl -L -o fregec.jar https://github.com/Frege/frege/releases/download/3.22.324/frege3.22.367-g2737683.jar
+
+script:
+  - make runtime 'shadow-prelude' compiler && make fregec.jar
```
Comment (tkawachi) [inline .travis.yml]: https://github.com/Frege/frege/wiki/Getting-Started doesn't mention about `shadow-prelude` target. Isn't it needed?

**Comment by Ingo60** [comment]
make runtime compiler fregec.jar

should be fine. 
Will we be able to access the built files? 
In that case, 

make runtime compiler fregec.jar dist

would be even better. 
Once this works, we will probably have to look how we can improve the getting the latest fregec.jar part.

**Comment by tkawachi** [comment]
If it's desirable to access the built files, travis support uploading to our own AWS S3 bucket which costs us a little.

CircleCI supports saving built files on their servers. I'll also try to build on CircleCI. then you can choose which to merge. (or both)

**Comment by tkawachi** [comment]
Please enable build from travis page. Thanks for merge.

**Comment by Ingo60** [comment]
It would be useful, but not indispensable, to be able to get the nightly build.

How do I enable this?

BTW, many thanks for caring.

**Comment by tkawachi** [comment]
Follow the instruction, step one and step two described at http://docs.travis-ci.com/user/getting-started/ to enable it. Then every push/pull requests will be compiled by travis :)

**Comment by Ingo60** [comment]
Yes, I will do that tomorrow. (It's already late in europe and I am not feeling like starting something new right now.)
Thank you again!

**Pull request merged**